### PR TITLE
[ADD] Added audio support in init

### PR DESCRIPTION
**Pull request by techpool**

### CODE REVIEW

File: src/util/data/list.js [hunk 1]
```diff
@@ -5,12 +5,14 @@ const
...
 	ListTuple = require('./../../entity/list')['ListTuple'],
-	PratilipiMetaDataListTuple = require('./../../entity/pratilipiMetaDataListTuple'),
+	PratilipiMetaDataListTuple = require('./../../entity/pratilipiMetaDataListTuple')['PratilipiMetaDataListTuple'],
+    PratilipiMetaDataListTupleV2 = require('./../../entity/pratilipiMetaDataListTuple')['PratilipiMetaDataListTupleV2'],
 	RecommendationType = require('./../../enum/recommendationType'),
 	Response = require('./../../entity/listApi'),
 	ASLAData = require('./../../entity/ASLAData'),
 	ASLACData = require('./../../entity/ASLACData'),
-	NewCategories = require('./../../enum/newCategories'),
+	NewCategories = require('./../../enum/newCategories')
```
Comment (sandeepm96) [inline src/util/data/list.js]: comma missing

File: src/enum/homepageSequence.js [hunk 2]
```diff
@@ -4,7 +4,8 @@ const Sequence = [
   { type: 'AUTHOR_LIST_MOST_ACTIVE' },
   { type: 'PRATILIPI_LIST', rows: -1 },    // -1 for the rest of the pratilipi lists left
   { type: 'VIDEO_LIST' },
-  { type: 'AUTHOR_LIST_TOP_RATED' }
+  { type: 'AUTHOR_LIST_TOP_RATED' },
+  { type: 'AUDIO_LIST' }
```
Comment (sandeepm96) [inline src/enum/homepageSequence.js]: @techpool position of audio list will be below 3 pratilipi lists after author list most active as per shally's requirement

so you can make changes to rows in line 5 and make it to 3 and insert audio list after that

File: src/util/data/list.js [hunk 3]
```diff
@@ -501,6 +503,9 @@ ListUtil.prototype._getCMSPratilipiMetaDataListTuple = async(listName, language,
 	// This takes care of cursor + offset + resultCount implementation
 	// cmsList will give cacheName in response => which will be included in cursor
 	const cmsList = await cmsDataUtil.getPratilipiList(listName, language, cacheName);
+    if (cmsList.listType !== CuratedListType.PRATILIPI_LIST.__name__) {
+        return null;
```
[the remark this answers is not in the document]
Comment (techpool) [reply src/util/data/list.js]: Made sure, all null is omitted at 
https://github.com/Pratilipi/ecs-init/blob/master/src/util/data/init.js#L97
https://github.com/Pratilipi/ecs-init/blob/master/src/util/data/init.js#L195
https://github.com/Pratilipi/ecs-init/blob/master/src/util/data/init.js#L287
https://github.com/Pratilipi/ecs-init/blob/master/src/util/data/init.js#L486

[the remark this answers is not in the document]
Comment (techpool) [reply src/util/data/list.js]: You can still take a look if I missed it somewhere internally.

[the remark this answers is not in the document]
Comment (sandeepm96) [reply src/util/data/list.js]: what about this line when `pratilipiMetaDataListTuple` is `null`: https://github.com/Pratilipi/ecs-init/blob/4779e994373810d835fc745e8d43afe023d1897a/src/util/data/list.js#L616

File: src/util/data/list.js [hunk 4]
```diff
@@ -501,6 +503,10 @@ ListUtil.prototype._getCMSPratilipiMetaDataListTuple = async(listName, language,
 	// This takes care of cursor + offset + resultCount implementation
 	// cmsList will give cacheName in response => which will be included in cursor
 	const cmsList = await cmsDataUtil.getPratilipiList(listName, language, cacheName);
+    if (cmsList.listType !== CuratedListType.PRATILIPI_LIST.__name__) {
+        let err = new Error();
+        err.data = Errors.LIST_NAME.ERR_LIST_TYPE_UNSUPPORTED;
```
Comment (sandeepm96) [inline src/util/data/list.js]: @techpool  throw err

**Pull request merged**